rubix-spi/src/main/java/com/qubole/rubix/spi/RetryingBookkeeperClient.java

```diff
@@ -88,6 +88,19 @@ public Void call() throws Exception
     });
   }
 
+  @Override
+  public boolean isBookKeeperAlive() throws TException
```

Comment:
No need to have retry logic here. We are already retrying in monit script if its not able to connect.